### PR TITLE
[microsoftteams] Updated microsoftTeams definitions to v.1.9.0

### PR DESCRIPTION
**Pull request by orty**
I would like to propose an update to the Microsoft Teams SDK (v.1.9.0) types definitions.
Just generated them using the provided d.ts generation script in the MS Teams SDK npm project source https://github.com/OfficeDev/microsoft-teams-library-js

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by orty** [comment]
Hi @WrathOfZombies @jayongg @ydogandjiev,

Kindly have a look at this PR please, I think I made it compliant with all mandatory checks but I might have missed something. Do not hesitate to let me know :)

File: types/microsoftteams/tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```
Comment (ydogandjiev) [inline types/microsoftteams/tsconfig.json:6]: Why is this change needed?

**Review by ydogandjiev — APPROVED** [review]

File: types/microsoftteams/tsconfig.json (hunk 1 above)
Comment (orty) [reply types/microsoftteams/tsconfig.json:6]: Hi @ydogandjiev,

Thanks for validating this PR, this change was requested because the new types contained definitions relative to the Window object.
